packages/frontend/src/presentation/components/button/Button.css

```diff
@@ -0,0 +1,18 @@
+.button {
+  display:block;
+  float:left;
+  margin:0 7px 0 0;
+  background-color:#f5f5f5;
```

Comment:
Entao....
Aqui comecamos a ter uns probleminhas...

Se formos usar CSS puro, precisaremos seguir uma boa convencao (BEM, Modular CSS or whatever) e ainda precisamos saber como lidar com reuso de estilos.
Por exemplo: Temos uma pasta com as constants, que expoem a nossa paleta de cor. Porem nao eh acessivel dentro do CSS. Para ser acessivel aqui precisariamos usar [Custom Properties](https://developer.mozilla.org/en-US/docs/Web/CSS/--*) que na real pode aumentar a complexidade do projeto.
Eu particularmente prefiro usar CSS puro, porem eu acho que vai aumentar o tempo que levamos para desenvolver as coisas. Talvez por isso poderiamos considerar usar Styled Components :/

Reply:
PS.: Nao sei quao bem Styled Component se comporta com TS

Reply:
Particularmente prefiro usar CSS puro no web e a api de styleSheet no native, styled component ja tenho mto aqui na firma kkk.

Reply:
Mas como vamos usar as cores compartilhadas?

Reply:
To pensando ainda, talvez usando sass?

Reply:
Sei nao. Mas a gente pode esperar o problema aparecer para resolver.